<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estrutura de Dados</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/Imagens/mainWindow.png</normaloff>:/Imagens/mainWindow.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_main">
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_Limpar">
        <property name="styleSheet">
         <string notr="true">border: 0px solid grey;
text-align: left; 

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_Selecao">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <item>
         <property name="text">
          <string>TUDO</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QListWidget" name="listWidget_MostrarLista">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/Imagens/PUCGO.png);</string>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>ESCOLHER</string>
    </property>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <action name="actionAbrir">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-Projeto_EstruturasDeDados-Desktop_Qt_5_6_0_MinGW_32bit-Debug/pesquisar.png</normaloff>../build-Projeto_EstruturasDeDados-Desktop_Qt_5_6_0_MinGW_32bit-Debug/pesquisar.png</iconset>
   </property>
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="iconText">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
